### PR TITLE
Add personal data info box

### PR DESCRIPTION
**Pull request by meldron**
I got the feedback,  that not everybody is comfortable to enter private/personal data in the Repo. This PR adds an info box that all personal data can be sent privately to one of the organizers.

### CODE REVIEW

File: src/routes/registration/+page.svelte [hunk 1]
```diff
@@ -24,6 +24,12 @@
 			</p>
 		</InfoBox>
 
+		<InfoBox title="Personal Data">
+			If you don't feel comfortable entering personal information publicly, 
+			please direct message one of the organizers on <a href="https://discord.gg/3mHhaXVEjP" title="JSCC Discord">Discord</a> 
+			and we will record information such as allergies or the like privately.
```
Comment (Narigo) [inline src/routes/registration/+page.svelte:30]:
```suggestion
			or reach out to <a href="mailto:team@jscraftcamp.org">team@jscraftcamp.org</a> and we will record information such as allergies or the like privately.
```

**Comment by wolframkriesing** [comment]
Why does using a nickname not cover that case?
We could also make `"name": "your name",` into `"name": "your name or nickname",` as we used to have it

**Comment by meldron** [comment]
> Why does using a nickname not cover that case? We could also make `"name": "your name",` into `"name": "your name or nickname",` as we used to have it

People were fine with the nick name argument for names, but the  `allergies` field was the main issue

**Comment by meldron** [comment]
> We could also make "name": "your name", into "name": "your name or nickname", as we used to have it

Very good idea, I will update the default template and registration page with that info.

**Pull request merged**